ray-operator/controllers/ray/rayservice_controller.go

```diff
@@ -1223,14 +1224,12 @@ func isServeAppUnhealthyOrDeployedFailed(appStatus string) bool {
 
 // TODO: Move this function to util.go and always use this function to retrieve the head Pod.
 func (r *RayServiceReconciler) getHeadPod(ctx context.Context, instance *rayv1.RayCluster) (*corev1.Pod, error) {
```

Comment:
Is it possible to only have a `getHeadPod` function that both the RayCluster controller and the RayService controller can call?

Reply:
Sure. I have moved it to `common.GetRayClusterHeadPod`.